src/main/init.c

```diff
@@ -21,6 +136,16 @@ int libre_init(void)
 {
 	int err;
 
+#if defined(HAVE_SIGNAL) && defined(SIGNAL_BTRACE)
+	(void)signal(SIGSEGV, signal_handler);
+	(void)signal(SIGABRT, signal_handler);
+	(void)signal(SIGILL, signal_handler);
+#endif
+
+#if defined(WIN32) && defined(SIGNAL_BTRACE)
+	SetUnhandledExceptionFilter(exception_handler);
+#endif
+
```

Comment:
should we make this optional in runtime?

another example: https://github.com/baresip/re/blob/main/src/sys/sys.c#L183